packages/charts/src/charts/bar/utils/chart.utils.ts

```diff
@@ -328,6 +328,7 @@ export const generateHorizontalStackedBars = ({
         width,
         height: barHeight,
         color: getPaletteColor(keys.indexOf(keyName), colors),
+        colorOutOfRange: !colors[index],
         value: data[index][keyName],
       };
 
```

Comment:
Additionally could You please modify or create additional story is Storybook that presents this use case.

Reply:
New story added